docs/pre.sh

```diff
@@ -1,41 +1,42 @@
 #!/usr/bin/env bash
 
 ## Create modules pages
-for D in ../x/*; do
+for D in x/*; do
   if [ -d "${D}" ]; then
-    MODDOC=build/modules/$(echo $D | awk -F/ '{print $NF}')
+    MODDOC=docs/docs/build/modules/$(echo $D | awk -F/ '{print $NF}')
```

Comment:
main on the SDK repo has only one https://github.com/cosmos/cosmos-sdk/tree/main/docs folder.
Have you tested in works on main?